### DISCUSSION TITLE
cms-wordpress-next example, hostname "secure.gravatar.com" is not configured under images in your next.config.js

### DISCUSSION THREAD

**Discussion by latonet** [vercel/next.js#32927] [Help]
https://github.com/vercel/next.js/tree/canary/examples/cms-wordpress

Did everything as per instructions, get this error at localhost ....

**Server Error**
Error: Invalid src prop (https://secure.gravatar.com/avatar/979a40d91439ef242d339c0717918b43?s=96&d=mm&r=g) on `next/image`, hostname "secure.gravatar.com" is not configured under images in your `next.config.js`
See more info: https://nextjs.org/docs/messages/next-image-unconfigured-host
This error happened while generating the page. Any console logs will be displayed in the terminal window.

**Comment by joaopereirawd** [comment]
Hey @latonet  try define  hostname `secure.gravatar.com` in your `next.config.js` such as: 
 
```
module.exports = {
    images: {
        domains: ['secure.gravatar.com'],
    },
}
```

**Reply by artypineda** [reply]
Thanks. This fixes: https://nextjs.org/examples > WordPress headless sample.

**Comment by amcc** [comment]
Got the same error. In Next 14 this worked for me. Is this the best way - and perhaps this should be the default if so:
```
/** @type {import('next').NextConfig} */
module.exports = {
  images: {
    remotePatterns: [
      {
        protocol: protocol.slice(0, -1),
        hostname,
        port,
        pathname: `${pathname}/**`,
      },
      {
        hostname: `*.gravatar.com`,
      },
    ],
  },
};
```

**Reply by caseyocampo** [reply]
This is what worked for me in March 2024, Next 14.
